<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>280</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>로그인</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_icon">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>210</width>
     <height>170</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_username">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pwd">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_registry">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>450</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
